ArduPlane/servos.cpp

```diff
@@ -205,15 +205,24 @@ void Plane::dspoiler_update(void)
     float dspoiler2_left = elevon_left;
     float dspoiler1_right = elevon_right;
     float dspoiler2_right = elevon_right;
+    RC_Channel *flapin = RC_Channels::rc_channel(g.flapin_channel-1);
```

Comment:
You need to check that a flapin channel was actually found. This could be nullptr.

Reply:
Unfortunately I'm useless after this point. I looked over another mix that used flap input and just took a couple of pieces from those lines and worked them into the differential spoilers mix and kept recompiling and flashing until everything moved the right directions.